desktop-widgets/downloadfromdivecomputer.cpp

```diff
@@ -117,10 +117,13 @@ DownloadFromDCWidget::DownloadFromDCWidget(const QString &filename, QWidget *par
 #define SETUPDC(num) \
 	if (!qPrefDiveComputer::vendor##num().isEmpty()) { \
 		ui.DC##num->setVisible(true); \
+		ui.DDC##num->setVisible(true); \
```

Comment:
`DDC` seems to be hard to understand, and introduces the potential for confusion with `DC` - why not be clear and use `DeleteDC`?